examples/karpenter-fargate/README.md

```diff
@@ -0,0 +1,210 @@
+# EKS Cluster with Karpenter Cluster Autoscaler running on Fargate
+
+Karpenter is an open-source node provisioning project built for Kubernetes. Karpenter automatically launches just the right compute resources to handle your cluster's applications. It is designed to let you take full advantage of the cloud with fast and simple compute provisioning for Kubernetes clusters.
+
+This example shows how to deploy and leverage Karpenter for Autoscaling and automatic nodes updating. Karpenter and other Add-ons are deployed on Fargate to avoid using any node groups. The Add-ons that cannot be deployed on Fargate (e.g. NTH) are deployed on nodes provisioned by Karpenter. The following resources will be deployed by this example.
+
+- VPC, 3 Private Subnets and 3 Public Subnets.
+- Internet gateway for Public Subnets and NAT Gateway for Private Subnets.
+- AWS EKS Cluster (control plane).
+- AWS EKS Fargate Profiles for the `kube-system` namespace with Pod Labels used by the `coredns`, `karpenter`, and `aws-load-balancer-controller` addons, while additional profiles can be added as needed.
+- AWS EKS managed addons `vpc-cni` and `kube-proxy`.
+- Karpenter Helm Chart.
+- AWS Node Termination Handler Helm Chart. Karpenter does not handle the Spot Interruption Termination Notice (ITN) two-minute warning. In lieu of this, you can install AWS Node Termination Handler to gracefully cordon and drain your spot nodes when they are interrupted. Pods that require checkpointing or other forms of graceful draining, requiring the 2-mins before shutdown, will need NTH.
```

Comment:
it does now in v0.19

Reply:
So should we wait for https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1145 to be complete an use it in this example?

Reply:
i expect to have the PR ready in a couple of days, so feel free to go ahead

Reply:
I see two options while we wait for the blueprint changes for Karpenter 0.19.0.

1. We specify that prior to v0.19.0, Karpenter requires NTH to handle the Spot ITN
2. We remove NTH from this example since it's not using Spot for the sample app

Reply:
Since Karpenter 0.20.0 is now available in the blueprints I removed NTH and configured Karpenter to use a SQS Interruption Queue.